README.md

```diff
@@ -15,7 +15,7 @@ repositories under the iPlantArchive organization, should you need them.
 The top-level directories __lein-plugins__, __libs__, __services__, __tools__,
 __databases__, __ui__, and __ansible__ are the categories that the code falls
 into. Each direct subdirectory contains a separate project. For instance,
-__services/Donkey/__ contains the service that the Discovery Environment UI
+__services/Terrain/__ contains the service that the Discovery Environment UI
```

Comment:
Minor, but do we want to retain the capitalized subdirectory? We don't have it for much else.